oat-sa/extension-tao-test-runner-plugins
# Fix/TR-2287/Get the autoPause option from the right config

Related to: https://oat-sa.atlassian.net/browse/TR-2287

The `blurPause` plugin detects when the user is looking outside of the browser window, thanks to the `blur` event. When this occurs, it has 2 different behaviours depending on an option:
- either it shows a warning
- either it pause the test

This option `sectionPause` condition this behaviour, and it is read from the test options by the plugin. However, the `testRunner.getOptions()` is taking the data from the `testData.config` and not from the `testContext.options` where the option is located at. This PR aims at fixing this by looking at both places.

How to test:
- install the extension
- activate the security feature
- add the category `x-tao-option-sectionPause` to any item of a test
- publish the delivery, and apply the security feature on it
- take the test, and try to go outside of the window (switch tab, click outside, open the dev tools, etc.)
- the test should be paused
